aries_cloudagent/connections/models/diddoc/service.py

```diff
@@ -32,15 +32,13 @@ class Service:
     everything else as URIs (oriented toward W3C-facing operations).
     """
 
+    """"""
+
     def __init__(
-        self,
-        did: str,
-        ident: str,
-        typ: str,
-        recip_keys: Union[Sequence, PublicKey],
-        routing_keys: Union[Sequence, PublicKey],
-        endpoint: str,
-        priority: int = 0,
```

Comment:
More specific args when possible would be preferable to condensing into *args and *kwargs

Reply:
Yes, I agree. Price to pay for using Marshmallow. With the new class I will try to do it more elegant.